agent-install/README.md

```diff
@@ -104,51 +104,40 @@ The script relies on an installation packages tree with the following directory
-    │   │   │       └── horizon_2.24.17~ppa~raspbian.buster_armhf.deb
-    │   │   └── stretch
-    │   │       └── armhf
-    │   │           ├── bluehorizon_2.24.17~ppa~raspbian.stretch_all.deb
-    │   │           ├── horizon-cli_2.24.17~ppa~raspbian.stretch_armhf.deb
-    │   │           └── horizon_2.24.17~ppa~raspbian.stretch_armhf.deb
-    │   └── ubuntu
-    │       ├── bionic
-    │       │   ├── amd64
-    │       │   │   ├── bluehorizon_2.24.17~ppa~ubuntu.bionic_all.deb
-    │       │   │   ├── horizon-cli_2.24.17~ppa~ubuntu.bionic_amd64.deb
-    │       │   │   └── horizon_2.24.17~ppa~ubuntu.bionic_amd64.deb
-    │       │   └── arm64
-    │       │       ├── bluehorizon_2.24.17~ppa~ubuntu.bionic_all.deb
-    │       │       ├── horizon-cli_2.24.17~ppa~ubuntu.bionic_arm64.deb
-    │       │       └── horizon_2.24.17~ppa~ubuntu.bionic_arm64.deb
-    │       │   
-    │       └── xenial
-    │           ├── amd64
-    │           │   ├── bluehorizon_2.24.17~ppa~ubuntu.xenial_all.deb
-    │           │   ├── horizon-cli_2.24.17~ppa~ubuntu.xenial_amd64.deb
-    │           │   └── horizon_2.24.17~ppa~ubuntu.xenial_amd64.deb
-    │           └── arm64
-    │               ├── bluehorizon_2.24.17~ppa~ubuntu.xenial_all.deb
-    │               ├── horizon-cli_2.24.17~ppa~ubuntu.xenial_arm64.deb
-    │               └── horizon_2.24.17~ppa~ubuntu.xenial_arm64.deb
-    │           
+    │   ├── debian
+    │   │   ├── buster
+    │   │   │   └── amd64
+    │   │   │       ├── horizon-cli_2.24.17~ppa~debian.buster_amd64.deb
+    │   │   │       └── horizon_2.24.17~ppa~debian.buster_amd64.deb
+    │   │   └── stretch
+    │   │       └── amd64
+    │   │           ├── horizon-cli_2.24.17~ppa~debian.stretch_amd64.deb
+    │   │           └── horizon_2.24.17~ppa~debian.stretch_amd64.deb
+    │   ├── raspbian
+    │   │   ├── buster
+    │   │   │   └── armhf
+    │   │   │       ├── horizon-cli_2.24.17~ppa~raspbian.buster_armhf.deb
+    │   │   │       └── horizon_2.24.17~ppa~raspbian.buster_armhf.deb
+    │   │   └── stretch
+    │   │       └── armhf
+    │   │           ├── horizon-cli_2.24.17~ppa~raspbian.stretch_armhf.deb
+    │   │           └── horizon_2.24.17~ppa~raspbian.stretch_armhf.deb
+    │   └── ubuntu
+    │       ├── bionic
+    │       │   ├── amd64
+    │       │   │   ├── horizon-cli_2.24.17~ppa~ubuntu.bionic_amd64.deb
+    │       │   │   └── horizon_2.24.17~ppa~ubuntu.bionic_amd64.deb
+    │       │   └── arm64
+    │       │       ├── horizon-cli_2.24.17~ppa~ubuntu.bionic_arm64.deb
+    │       │       └── horizon_2.24.17~ppa~ubuntu.bionic_arm64.deb
+    │       └── xenial
+    │           ├── amd64
+    │           │   ├── horizon-cli_2.24.17~ppa~ubuntu.xenial_amd64.deb
+    │           │   └── horizon_2.24.17~ppa~ubuntu.xenial_amd64.deb
+    │           └── arm64
+    │               ├── horizon-cli_2.24.17~ppa~ubuntu.xenial_arm64.deb
+    │               └── horizon_2.24.17~ppa~ubuntu.xenial_arm64.deb
```

Comment:
This is not correct. The new directory structure that `agent-install.sh` requires is a flat directory of files, like:

```
agent-install.cfg
agent-install.crt
agent-install.sh
agent-uninstall.sh
amd64_anax.tar.gz
amd64_anax_k8s.tar.gz
deployment-template.yml
horizon-2.27.0-160.x86_64.rpm
horizon-cli-2.27.0-160.pkg
horizon-cli-2.27.0-160.x86_64.rpm
horizon-cli.crt
horizon-cli_2.27.0-160_amd64.deb
horizon-cli_2.27.0-160_arm64.deb
horizon-cli_2.27.0-160_armhf.deb
horizon_2.27.0-160_amd64.deb
horizon_2.27.0-160_arm64.deb
horizon_2.27.0-160_armhf.deb
persistentClaim-template.yml
```

(Not all files are required to be present, just the files that are needed for that edge node type.)

Reply:
@bmpotter,  It is updated, please help review that again, Thanks